src/content/en/2021/compression.md

```diff
@@ -23,36 +23,36 @@ unedited: true
 
 ## HTTP compression is useful for many types of content
 
-HTTP compression is recommended for most text-based content, such as [HTML](./markup), [CSS](./css), [JavaScript](./javascript), JSON, or SVG. It's also recommended for certain non-text formats such as woff, ttf, ico because they still benefit from lossless compression. On the other hand, for media files such as images that are already compressed it doesn't provide a benefit because compressing already-compressed files doesn't reduce their size much and may even make them slightly larger.
+We recommend HTTP compression for text-based content, such as [HTML](./markup), [CSS](./css), [JavaScript](./javascript), JSON, or SVG, as well as for woff, ttf and ico. Media files such as images that are already compressed do not benefit from HTTP compression since their representation already includes internal compression.
 
 {{ figure_markup(
   image="compession-methods-by-content-type.png",
   caption="Compression methods for different content types",
-  description="A stacked bar chart showing the usage rate of different compression algorithms broken down by the content type. The stacked bars divide up the use of Brotli, Gzip, and no compression. `text/html` is the only content type that is compressed less than 50% of the time. `application/json` is approximately 68% compressed, `image/svg+xml` approximately 64%. `text/css` and `application/javascript` are each over 85% compressed. `application/x-javascript` and `text/javascript` are greater than 90% compressed.",
+  description="A stacked bar chart showing the usage rate of different compression algorithms broken down by the content type. The stacked bars divide up the use of Brotli, Gzip, and no compression. `text/plain` and `text/html` are the only content types that are compressed less than 50% of the time. `application/json` is compressed approximately 68% of the time, `image/svg+xml` approximately 64%. `text/css` and `application/javascript` are each compressed over 85% of the time, and `application/x-javascript` and `text/javascript` are compressed more than 90% of the time.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQtfyTM9VEweN_Hli3IuxxqU1CRap4V5Q28baEs7aEBResoPRgk9Dwp1m_vdS9lzNlfO8J4hZN7GPT7/pubchart?oid=1292728213&format=interactive",
   sheets_gid="234112121",
   sql_file="compression_by_content_type.sql"
   )
 }}
 
-Compared to the other content type, `text/plain` and `text/html` uses the least amount of compression, with merely 12% and 14% using compression at all. This might be because `text/html` is more often dynamically generated than static content such as JavaScript and CSS.
+Compared to the other content types, `text/plain` and `text/html` uses the least amount of compression, with merely 12% and 14% using compression at all. This might be because `text/html` is more often dynamically generated than static content such as JavaScript and CSS.
```

Comment:
I added: `However, it is recommended to compress dynamic content as well, and the next sections show how to configure this.`

Because the next section gives the distinction between dynamic and static content, and then further down the section about how to improve compression gives the recommendation of levels to use for dynamic and for static content